examples/shouldi/README.md

```diff
@@ -29,9 +29,15 @@ Right now `shouldi` runs the following static analysis tools and complains if:
 
 - [safety](https://pyup.io/safety/)
   - Any issues are found
-- TODO: [bandit](https://pypi.org/project/bandit/)
+  
 - TODO: [pylint](https://pypi.org/project/pylint/)
   - TDB (something about the number of errors)
+  
+Also , shouldi as been updated with following code wrappers:
```

Comment:
- I don't think we need to say `Also, ...` here. We can just list all of them.
- Let's also remove the `Any issues are found` line below the `safety` bullet point

Reply:
OK I am working on it

Reply:
Done